ckanext/privatedatasets/actions.py

```diff
@@ -133,9 +134,10 @@ def package_deleted(context, request_data):
     :rtype: dict
 
     '''
-    return _process_package(context, request_data, 'delete')
+    context += {'method': 'disallow'}
```

Comment:
I commited the changes to pull them from my other OS and test the changes.
No problem, let me test it and I'll push it with 'revoke'